### DISCUSSION TITLE
Is there a way to background/drop to a shell.

### DISCUSSION THREAD

**Discussion by chrhlnd** [helix-editor/helix#1931] [Q&A]
Used to hitting ctrl+z a lot. Wondering if there is a way to hop into a new session of the shell hx was started with.

Hmm I'm expecting this to work on windows. Maybe it does on linux.

**Comment by sudormrfbin** [comment]
`Ctrl-z` _is_ supported but IIRC there were some minor issues with it which @pickfire debugged and found the cause for. Not sure if it was fixed, not sure about the windows support either.

**Reply by pickfire** [reply]
`ctrl-z` works but in some cases it don't work, like if you are editing git commit messages with helix and suspend then it will get stuck, haven't figured out that issue yet. But if you started helix from shell it should work, I only use linux so not sure if it works on windows, but the code path is only built for linux and not sure if windows have similar solution, but if you use WSL then it should work with the linux binary.
